yt/go/yt/integration/auth_client_test.go

```diff
@@ -65,6 +68,22 @@ func (s *Suite) TestIssueListRevokeToken(ctx context.Context, t *testing.T, yc y
 	require.Empty(t, tokens)
 }
 
+func (s *Suite) TestWhoAmI(ctx context.Context, t *testing.T, yc yt.Client) {
+	user := "user-" + guid.New().String()
+	_ = s.CreateUser(ctx, t, user)
+	token, err := yc.IssueToken(ctx, user, "", nil)
+	require.NoError(t, err)
+	require.NotEmpty(t, token)
+
+	userClient, err := ythttp.NewClient(&yt.Config{Proxy: os.Getenv("YT_PROXY"), Logger: s.Env.L, Token: token})
```

Comment:
Offtop, but I don't like this idea of passing proxy for test in env var, but I see that the current test run configuration doesn't have another way for it.

Reply:
Yes, I agree that tis looks ugly, but I copied this method from the client creation in other testing methods.